core/bus/bus.go

```diff
@@ -95,6 +106,14 @@ func (b *Bus) SendEvent(queues []string, ev Event) {
 	b.lock.Lock()
 	defer b.lock.Unlock()
 
+	if _, ok := b.events[ev.Event]; !ok {
+		b.events[ev.Event] = 0
+	}
+	if _, ok := b.messages[ev.Event]; !ok {
+		b.messages[ev.Event] = 0
+	}
```

Comment:
These initializations are actually unnecessary, as referencing a non-existent map entry returns its zero value. `b.events[iDontExist]++` would set the value to 1 without issue. It's up to you if you want to keep the initialization though. It's not hurting anything.

Reply:
I see the purpose now in the DumpState loop, so scratch this.